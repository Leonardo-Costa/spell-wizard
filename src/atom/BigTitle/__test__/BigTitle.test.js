import React from 'react';
import renderer from 'react-test-renderer';

import {BigTitle} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<BigTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });