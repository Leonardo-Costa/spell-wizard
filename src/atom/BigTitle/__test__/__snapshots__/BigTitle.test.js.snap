// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#202020",
        "fontSize": 60,
        "fontWeight": "bold",
      }
    }
  />
</View>
`;
